packages/components/src/Well/_docs/Well.stories.tsx

```diff
@@ -3,12 +3,14 @@ import { Meta, StoryObj } from "@storybook/react"
 import { Heading } from "~components/Heading"
 import { Text } from "~components/Text"
 import { Well } from "../index"
-import { borderStyleTypes, variantTypes } from "../types"
+import { borderStyleTypes } from "../types"
 
 const meta = {
   title: "Components/Well",
   component: Well,
   args: {
+    color: "white",
+    borderStyle: "solid",
```

Comment:
Given these have default values, it might actually be better to not declare these so that we can see what happens when they are `undefined`

Reply:
True, will remove them :)